specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
The election definitions that we need (I think) are all in the CVR column header lines, e.g. `Presidential Electors (Vote For=1)`.